README.md

```diff
@@ -57,41 +60,103 @@ If you are interested in becoming a member of the team check out the **[Pixelogi
+
+1. Copy `template_pushMaster.yml` file from the workflows template directory, `.github/workflow_templates/`, into the workflows directory, `.github/workflows`.
+1. Rename the file copied with the function's name. E.g. function `tokengen` would be renamed: `tokengen_pushMaster.yml`
+1. There are three lines to update in the file with the function's name: 1, 8, & 17
+    - **line 1**: Replace [function name] with the function's name (e.g. `tokengen`)
+    - **line 8**: Replace [function name] with the function's name (e.g. `tokengen`)
+    - **line 17**: Replace [function name] with the function's name (e.g. `tokengen`)
+1. That is all, GitHub Actions is now configured for the function. (after the files are pushed to master that is 😃)
+
+### Deployment Status (refresh page to update)
+
+![appleauth function push master](https://github.com/PixelogicDev/gruveebackend/workflows/appleauth%20function%20push%20master/badge.svg)  
+![createappledevtoken function push master](https://github.com/PixelogicDev/gruveebackend/workflows/createappledevtoken%20function%20push%20master/badge.svg)  
+![createprovideruser function push master](https://github.com/PixelogicDev/gruveebackend/workflows/createprovideruser%20function%20push%20master/badge.svg)  
+![createsocialplaylist function push master](https://github.com/PixelogicDev/gruveebackend/workflows/createsocialplaylist%20function%20push%20master/badge.svg)  
+![createuser function push master](https://github.com/PixelogicDev/gruveebackend/workflows/createuser%20function%20push%20master/badge.svg)  
+![doesuserdocexist function push master](https://github.com/PixelogicDev/gruveebackend/workflows/doesuserdocexist%20function%20push%20master/badge.svg)  
+![getspotifymedia function push master](https://github.com/PixelogicDev/gruveebackend/workflows/getspotifymedia%20function%20push%20master/badge.svg)  
+![socialplatform function push master](https://github.com/PixelogicDev/gruveebackend/workflows/socialplatform%20function%20push%20master/badge.svg)  
+![socialtokenrefresh function push master](https://github.com/PixelogicDev/gruveebackend/workflows/socialtokenrefresh%20function%20push%20master/badge.svg)  
+![spotifyauth function push master](https://github.com/PixelogicDev/gruveebackend/workflows/spotifyauth%20function%20push%20master/badge.svg)  
+![tokengen function push master](https://github.com/PixelogicDev/gruveebackend/workflows/tokengen%20function%20push%20master/badge.svg)  
+![updatealgolia function push master](https://github.com/PixelogicDev/gruveebackend/workflows/updatealgolia%20function%20push%20master/badge.svg)  
+![usernameavailable function push master](https://github.com/PixelogicDev/gruveebackend/workflows/usernameavailable%20function%20push%20master/badge.svg)
+
```

Comment:
This is SICK.

I feel like we should add this to a separate README file that just has this info. Also could we add to the instructions that for every new function, we need to update this?

Reply:
Are you suggesting only the badges in another readme or the deployment stuff as a whole?

Reply:
updated instructions in last commit

Reply:
No I am referring to this entire Deployment status section. More specifically:

In `README-Support` it would be cool to add a "Deployment Status" page and then in the main README section reference that you can get deployment status in that section.

Let me know if this makes sense and your thoughts on it

Reply:
@adilanchian Pushed up a new commit. Thoughts on this approach?